janus_core/calculations/md.py

```diff
@@ -282,13 +275,33 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals,too-many-sta
             Random seed used by numpy.random and random functions, such as in Langevin.
             Default is None.
         """
+        [
```

Comment:
Prefer round braces as this is to me tuple unpacking with a fixed set or an operation rather than a variable list.